src/main/java/com/google/sps/data/Pair.java

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/**
+* Represents two objects.
+*/
+public class Pair<Type1, Type2> {
```

Comment:
Can the "Map.Entry" service the purpose?
https://docs.oracle.com/javase/8/docs/api/java/util/Map.Entry.html

Reply:
Probably, but I think a Pair is clearer in that it doesn't necessarily consider the first element as a "key" and the second element as a "value". This sort of naming could be confusing if the relationship between the two elements is not like that. Also see https://github.com/googleinterns/step55-2020/pull/8#discussion_r447739305.

Reply:
Got it, thanks!